unit-tests/src/test/java/org/eclipse/collections/api/block/predicate/PredicateTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 The Bank of New York Mellon.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.api.block.predicate;
+
+import org.eclipse.collections.impl.block.factory.Predicates;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PredicateTest
+{
+    @Test
+    public void test()
+    {
+        Predicates<Object> alwaysTrue = Predicates.alwaysTrue();
+        Assert.assertTrue(alwaysTrue.test(Boolean.TRUE));
+        Assert.assertTrue(alwaysTrue.test(Boolean.FALSE));
+
+        Predicates<Object> alwaysFalse = Predicates.alwaysFalse();
```

Comment:
Change the type from `Predicates` to Predicate, otherwise this is a `PredicatesTest`, not `PredicateTest`.

Reply:
Fixed